tool/tsh/tsh.go

```diff
@@ -420,6 +423,8 @@ func Run(args []string, opts ...cliOption) error {
 	app.Flag("proxy", "SSH proxy address").Envar(proxyEnvVar).StringVar(&cf.Proxy)
 	app.Flag("nocache", "do not cache cluster discovery locally").Hidden().BoolVar(&cf.NoCache)
 	app.Flag("user", fmt.Sprintf("SSH proxy user [%s]", localUser)).Envar(userEnvVar).StringVar(&cf.Username)
+	app.Flag("memprofile", "Write memory profile to file").StringVar(&memProfile).Hidden()
+	app.Flag("cpuprofile", "Write CPU profile to file").StringVar(&cpuProfile).Hidden()
```

Comment:
👍 very handy, thanks for adding them.

nit: personal preference `cpu-profile` vs `cpuprofile`. I feel likely I would type `--cpu-profile` unconsciously as other flags have dashes dividing the words.

Reply:
Sure, happy to do that. These names are just from the canonical Go example, nothing I'm tied to.